src/spikeinterface/sortingcomponents/matching/method_list.py

```diff
@@ -12,3 +12,11 @@
     "circus-omp-svd": CircusOMPSVDPeeler,
     "wobble": WobbleMatch,
 }
+
+try:
+    # kilosort licence (GPL 3) is forcing use to make an external package
+    from spikeinterface_kilosort_reshape import KiloSortMatching
```

Comment:
really think that `spikeinterface_kilosort_reshape` is a bad name. @samuelgarcia let's use `spikeinterface_kilosort_components`